apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet/Trace/SimpleManagedTracer.cs

```diff
@@ -19,7 +19,9 @@
 using System.Collections.Generic;
 using System.Diagnostics;
 
-namespace Google.Devtools.AspNet
+using CloudTrace = Google.Cloud.Trace.V1.Trace;
```

Comment:
Why do we need this now? If you just have a using directive for `Google.Cloud.Trace.V1`, can't you just use `Trace`? I haven't got my head round the conflict here.